projectdoc/implementation.tex

```diff
@@ -59,7 +59,17 @@ \subsubsection{Plaza Vorverarbeitung}
     \item Nun werden alle in den vorherigen Schritten erstellten temporären \ac{OSM}-Dateien mit der originalen Datei zusammen geschmolzen, die ursprünglich importiert wurde. Dazu wird das externe Tool Osmosis \cite{osmosis} aufgerufen.
 \end{enumerate}
 
-Als Endresultat liegt nun eine \ac{OSM}-Datei mit den kompletten Kartendaten vor, die um zusätzliche Linien für die Optimierung über Fussgängerflächen ergänzt wurde. Die Datei kann nun etwa einer Routing-Engine übergeben werden.
+Als Endresultat liegt nun eine \ac{OSM}-Datei mit den kompletten Kartendaten vor, die um zusätzliche Linien für die Optimierung über Fussgänger-Flächen ergänzt wurde. Die Datei kann nun etwa einer Routing-Engine übergeben werden.
+
+\paragraph{Konfiguration}\label{impl:preprocess_konfiguration}~\\
```

Comment:
@Excape hier würde ich kurz in einem Absatz noch sagen, dasss man das bei Hindernissen auch machen kann

Reply:
done